oc-chef-pedant/spec/api/controls_spec.rb

```diff
@@ -12,419 +12,432 @@
-
-    context 'POST /controls' do
-      after :each do
-        delete(api_url("clients/client.opscode.piab"), platform.superuser)
-        delete("#{platform.server}/clients/client.opscode.piab", platform.superuser)
-
-        delete(api_url("clients/clientwithnonode.opscode.piab"), platform.superuser)
-        delete("#{platform.server}/clients/clientwithnonode.opscode.piab", platform.superuser)
-      end
-
-      it 'returns a 201 ("Created") for client' do
-        post(api_url("/controls"),
-            client,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 201
-              })
-            end
-      end
-
-      it 'returns a 403 ("Forbidden") for normal_user' do
-        post(api_url("/controls"),
-            normaluser,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 403
-              })
-            end
-      end
-
-      it 'returns a 403 ("Forbidden") for admin_user' do
-        post(api_url("/controls"),
-            adminuser,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 403
-              })
-            end
-      end
-
-    end
-
-    context 'GET /controls' do
-      after :each do
-        delete(api_url("clients/client.opscode.piab"), platform.superuser)
-        delete("#{platform.server}/clients/client.opscode.piab", platform.superuser)
-
-        delete(api_url("clients/clientwithnonode.opscode.piab"), platform.superuser)
-        delete("#{platform.server}/clients/clientwithnonode.opscode.piab", platform.superuser)
-
-      end
-
-      it 'returns a 405 ("Forbidden") for client' do
-        get(api_url("/controls"),
-            client,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 405
-              })
-            end
-      end
+      let(:client){ platform.create_client('client.opscode.piab') }
+      let(:clientwithnonode){ platform.create_client('clientwithnonode.opscode.piab') }
+      let(:normaluser){ normal_user }
+      let(:adminuser){ admin_user }
+
+      context 'POST /controls' do
+        after :each do
+          delete(api_url("clients/client.opscode.piab"), platform.superuser)
+          delete("#{platform.server}/clients/client.opscode.piab", platform.superuser)
+
+          delete(api_url("clients/clientwithnonode.opscode.piab"), platform.superuser)
+          delete("#{platform.server}/clients/clientwithnonode.opscode.piab", platform.superuser)
+        end
 
-      it 'returns a 405 ("Method Not Allowed") for normal_user' do
-        get(api_url("/controls"),
-            normaluser,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 405
-              })
-            end
-      end
+        it "returns a 201 ('Created') for client" do
```

Comment:
What's the reasoning behind changing the string formatting here? It's generally accepted good Ruby style to use single quotes `'foo'` for strings that don't need in interpolation and double quotes `"#{foo}"` for strings that do need interpolation. That appeared to be generally true here, and I just wanted to see if there was a reason for moving away from it.

Having said that, there's no clear performance advantage to either and it's more for consistency than anything else. I'll also admit that there's not much consistency in this file to begin with (see following line), so I'm really just raising a point that probably doesn't need much follow up, unless you wanted to enforce consistency on the whole file :)